css/properties/-moz-image-region.json

```diff
@@ -18,10 +18,10 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": true
+              "version_added": "54"
```

Comment:
I'm still unclear on these two. The PR description suggests an early version (1/4), not 54

Reply:
...oops.  That was meant to be set to “1”.  Fixing now!